### PR TITLE
added basic , leading and stacked snackbars

### PR DESCRIPTION
**Pull request by atharvabhanage02**
Added Snackbar : 

- Baseline Snackbar
- Leading Snackbar
- Stacked Snackbar

Here is the Preview : https://deploy-preview-23--apex-ui-p1.netlify.app/documentation/docs

### CODE REVIEW

**Review by gopalbbharadva — APPROVED** [review]
Component loos good and working fine, good to go.

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+.snackbar-buttons {
+  margin-top: 2.5rem;
+  display: flex;
+  justify-content: space-evenly;
+}
+.btn-snackbar {
+  color: var(--txt-grey-100);
+  font-weight: bold;
+  padding: 10px;
```
Comment (SomeEarth86) [inline components/snackbar/snackbar.css:16]: suggestion/ refactor later : keep CSS units consistent ( if using rem use it across all the places)

reference: https://nekocalc.com/px-to-rem-converter

**Review by SomeEarth86 — APPROVED** [review]
Preview looks good, Code is good overall.

good to merge.

File: components/snackbar/snackbar.css (hunk 1 above)
Comment (atharvabhanage02) [reply components/snackbar/snackbar.css:16]: Thanks for suggesting the resource , will surely look into it

**Pull request merged**